rex/utilities/execution.py

```diff
@@ -444,6 +457,95 @@ def scancel(job_id):
         cmd = shlex.split(cmd)
         call(cmd)
 
+    @staticmethod
+    def scancel_all():
+        """Cancel ALL user jobs"""
+        sq = SLURM.squeue()
+        for row in sq[1:]:
+            job_id = row.strip().split(' ')[0]
+            SLURM.scancel(job_id)
+
+    @staticmethod
+    def change_qos(job_id, qos):
+        """Change the priority (quality of service) for a job.
+
+        Parameters
+        ----------
+        job_id : int
+            SLURM job id to cancel
+        qos : str
+            New qos value
+        """
+
+        cmd = 'update job {} QOS={}'.format(job_id, qos)
+        SLURM.scontrol(cmd)
+
+    @staticmethod
+    def change_qos_all(qos):
+        """Change the priority (quality of service) for all jobs.
+
+        Parameters
+        ----------
+        qos : str
+            New qos value
+        """
+        sq = SLURM.squeue()
+        for row in sq[1:]:
+            row_list = [x for x in row.strip().split(' ') if x != '']
+            job_id = row_list[0]
+            status = row_list[4]
+            if status == 'PD':
+                SLURM.change_qos(job_id, qos)
+
+    @staticmethod
+    def hold(job_id):
+        """Temporarily hold a job from submitting.
+
+        Parameters
+        ----------
+        job_id : int
+            SLURM job id to cancel
+        """
+
+        cmd = 'hold {}'.format(job_id)
+        SLURM.scontrol(cmd)
+
+    @staticmethod
+    def hold_all():
+        """Hold all pending jobs from submitting"""
+        sq = SLURM.squeue()
+        for row in sq[1:]:
+            row_list = [x for x in row.strip().split(' ') if x != '']
+            job_id = row_list[0]
+            status = row_list[4]
+            if status == 'PD':
+                SLURM.hold(job_id)
+
+    @staticmethod
+    def release(job_id):
```

Comment:
I don't under stand what this means?

Reply:
Tried to clarify the doc strings for hold and release.